.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Should .vscode files be added to the git ignore?

Reply:
Ah, yep.  Let me clean that up.